UnleashClient/constraints/Constraint.py

```diff
@@ -139,8 +144,16 @@ def check_date_operators(self, context_value: Union[datetime, str]) -> bool:
 
         try:
             parsed_date = parse(self.value)
+
+            # If parsed date is timezone-naive, assume it is UTC
+            if parsed_date.tzinfo is None:
```

Comment:
This originates on the server and I don't think this should ever happen. Is this necessary?